centreon/packages/ui/test/setupTests.js

```diff
@@ -4,42 +4,42 @@
 // import '@testing-library/jest-dom';
 // import '@testing-library/react';
 // import '@testing-library/jest-dom/extend-expect';
-import i18n from "i18next";
-import { initReactI18next } from "react-i18next";
-import fetchMock from "jest-fetch-mock";
+import i18n from 'i18next';
```

Comment:
Use double-quotes here

Reply:
😅 iv just changed them all to single quotes in the previous PR, as this is what the linter pre-scribes